ViewRecipe.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+
+
+
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.3.0/font/bootstrap-icons.css">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+ <link rel="stylesheet" href="showRecipeStyle.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Marwan</title>
+</head>
+
+<body>
+    <div class="container p-3">
+  
+    <!-- Show recipe -->
+    <!-- change them to Dynamic -->
+    <div class="row">
+        <div class="col-12 mb-5">
+            <h1 class="h1 text-center" >Recipe title</h1>
+        </div>
+        <div class="col-12 showForm d-flex">
+            <div class="col-5 imageDiv">
+                <img   src="img/dish4.jpg"  alt="recipe_image"     class="recipe_image"/>
+            </div>
+            <!-- ingredients Div -->
+            <div class="col-7">
+                <table class="table table-striped ingredientsDiv">
+                    <thead>
+                        <tr>
+                            <th scope="col">ID </th>
+                            <th scope="col">Ingredients </th>
+            
+                        </tr>
+                    </thead>
+                    <tbody>
+                        <tr>
+                            <th scope="row">1</th>
+                            <td> 1 flat iron steak</td>
+            
+                        </tr>
+                        <tr>
+                            <th scope="row">2</th>
+                            <td>Red onion, chopped</td>
+            
+                        </tr>
+                        <tr>
+                            <th scope="row">4</th>
+                            <td>cloves garlic, minced</td>
+            
+                        </tr>
+                        <tr>
+                            <th scope="row">5</th>
+                            <td>2 pinches salt and coarsely ground pepper</td>
+            
+                        </tr>
+                        <tr>
+                            <th scope="row">3</th>
+                            <td>2 tablespoons Worcestershire sauce</td>
+            
+                        </tr>
+                    </tbody>
+                </table>
```

Comment:
You are showing the same ingredients no matter on which recipe I click 'View Recipe'. Instead, it should work in the way that when I click it, it shows the matching recipe. I would recommend having empty td elements to start with and then dynamically add the matching data when the 'View Recipe button is clicked. 😊